### PR TITLE
Connect followers and following with Leleka's backend

### PR DESCRIPTION
**Pull request by vasyagladush**

### CODE REVIEW

**Comment by alexandrtovmach** [comment]
conflicts

File: src/containers/FollowingList.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+              data={following}
+              itemComponent={(itemData) => (
+                <LiWrapper>
+                  <FollowerUserItem key={itemData._id} user={itemData} />
+                </LiWrapper>
+              )}
+            />
+          </ListGroup>
+        ) : (
+          "You don't follow anyone"
```
Comment (KirillSerg) [inline src/containers/FollowingList.tsx]: i18n?

File: src/containers/FollowingList.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        )
+      ) : (
+        "You are not authorized"
```
Comment (KirillSerg) [inline src/containers/FollowingList.tsx]: i18n?

File: src/containers/FollowersList.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+              data={followers}
+              itemComponent={(itemData) => (
+                <LiWrapper>
+                  <FollowerUserItem key={itemData._id} user={itemData} />
+                </LiWrapper>
+              )}
+            />
+          </ListGroup>
+        ) : (
+          "You have no followers"
```
Comment (KirillSerg) [inline src/containers/FollowersList.tsx]: i18n?

File: src/containers/FollowersList.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        )
+      ) : (
+        "You are not authorized"
```
Comment (KirillSerg) [inline src/containers/FollowersList.tsx]: i18n?

**Review by KirillSerg — APPROVED** [review]
good for me

**Pull request merged**